ledger/accountdb.go

```diff
@@ -1130,26 +1124,15 @@ func resetCatchpointStagingBalances(ctx context.Context, tx *sql.Tx, newCatchup
 // tables and update the correct balance round. This is the final step in switching onto the new catchpoint round.
 func applyCatchpointStagingBalances(ctx context.Context, tx *sql.Tx, balancesRound basics.Round, merkleRootRound basics.Round) (err error) {
 	stmts := []string{
-		"ALTER TABLE accountbase RENAME TO accountbase_old",
-		"ALTER TABLE assetcreators RENAME TO assetcreators_old",
-		"ALTER TABLE accounthashes RENAME TO accounthashes_old",
-		"ALTER TABLE resources RENAME TO resources_old",
-		"ALTER TABLE onlineaccounts RENAME TO onlineaccounts_old",
-		"ALTER TABLE txtail RENAME TO txtail_old",
+		"DROP TABLE IF EXISTS accountbase",
```

Comment:
did you test it and tables survive after rolling back the transaction?

Reply:
just tested it